### PR TITLE
Fix font sizes for ALL PAGES

### PR DESCRIPTION
**Pull request by zzhikai**

### CODE REVIEW

File: src/components/styles/Button.styled.ts [hunk 1]
```diff
@@ -6,7 +6,7 @@ export const BaseButton = styled.div<{ fontSize?: string; isActive?: boolean }>`
   align-items: center;
   .ant-btn-text {
     color: ${(props) => props.theme.palette.common.white};
-    font-size: ${(props) => props.fontSize};
+    font-size: ${(props) => `clamp(14px, 2vw, ${props.fontSize})`};
```
Comment (hui444) [inline src/components/styles/Button.styled.ts:9]: might want to rename max font size?

Comment (hui444) [reply src/components/styles/Button.styled.ts:9]: or use themes for this

File: src/styles/Theme.tsx [hunk 2]
```diff
@@ -16,58 +16,58 @@ const theme: DefaultTheme = {
...
+        max: '18px',
         weight: '300',
         height: '',
       },
       input: {
-        min: '24px',
-        size: '36px',
-        max: '2.4rem',
+        min: '16px',
+        size: '1.25vw',
```
Comment (hui444) [inline src/styles/Theme.tsx:27]: values here should be scale according to the range; ie if 1.25vw for 12px then for 16px it should be something higher than 1.25, can be like 1.27 etc; should scale too

File: src/components/styles/Button.styled.ts (hunk 1 above)
Comment (zzhikai) [reply src/components/styles/Button.styled.ts]: 6f83ddcef630951d836d70309f969b0953101e6e

File: src/styles/Theme.tsx (hunk 2 above)
Comment (zzhikai) [reply src/styles/Theme.tsx]: 6f83ddcef630951d836d70309f969b0953101e6e

**Comment by hui444** [comment]
lgtm!

**Review by hui444 — APPROVED** [review]
lgtm!

**Pull request merged**